cmd/psa/test_common.go

```diff
@@ -108,27 +107,13 @@ var (
 		  }`)
 )
 
-func makeClaimsFromJSON(j []byte, partial bool) psatoken.IClaims {
-	p2 := &psatoken.P2Claims{}
-
-	err2 := json.Unmarshal(j, p2)
-	if err2 == nil {
-		if partial || p2.Validate() == nil {
-			return p2
-		}
+func makeClaimsFromJSON(j []byte, validate bool) psatoken.IClaims {
+	claims, err := claimsFromJSON(j, validate)
+	if err != nil {
```

Comment:
this is used in testing, so instead of panic'ing you should do like mustHexDecode, which gets passed the testing.T object and calls the appropriate `require` method on failure.

Reply:
Tracked in #8